tests/baselines/reference/switchStatementsWithMultipleDefaults.errors.txt

```diff
@@ -26,17 +22,15 @@ tests/cases/compiler/switchStatementsWithMultipleDefaults.ts(27,22): error TS110
             switch (x * x) {
                 default:    // No issues.
                 default:    // Error; second 'default' clause.
-                ~~~~~~~~
-!!! error TS1113: A 'default' clause cannot appear more than once in a 'switch' statement.
                     break;
                 case 10000:
                     x /= x;
                 default:    // Error, third 'default' clause
                 def\u0061ult: // Error, fourth 'default' clause.
+                ~~~~~~~~~~~~
+!!! error TS1260: Keyword must not contain escaped characters

```

Comment:
Yep, it is correct to issue an error here. You can paste `const a = {def\u0061ult: 12}` into the repl and be greeted with an error. It surprised me, tbh.

Reply:
Fun fact: After some discussion with @rbuckton, while this errors in chrome (therefore v8), it does not in firefox and edge - per spec it doesn't seem like it _should_ be an error, either (since anywhere an identifier is parse-able it should work). [A crbug has been logged](https://bugs.chromium.org/p/chromium/issues/detail?id=993000&can=2&q=unicode%20escape)

Reply:
@rbuckton checked jsc as well, which matches chrome's behavior - [I've opened a bug there, too.](https://bugs.webkit.org/show_bug.cgi?id=200638)

Reply:
I dug through test262 but couldn't find a test for this case. The closest I could find was for a BindingIdentifier, not an IdentifierName:
- https://github.com/tc39/test262/blob/master/test/language/identifiers/val-default.js
- https://github.com/tc39/test262/blob/master/test/language/identifiers/val-default-via-escape-hex4.js (Unicode escapes in BindingIdentifier)
- https://github.com/tc39/test262/blob/master/test/language/reserved-words/ident-name-keyword-prop-name.js